command/server.go

```diff
@@ -458,6 +458,11 @@ func (c *ServerCommand) runRecoveryMode() int {
 		JSONFormat: logFormat == logging.JSONFormat,
 	})
 
+	// reporting Errors found in the config
+	for _, cErr := range configErrors {
+		c.logger.Error(cErr.String())
```

Comment:
I used `Error` log level here. Should it be `warning` instead?

Reply:
I would warn, yes, because the server may be able to continue.